app/ante/eth.go

```diff
@@ -322,8 +315,6 @@ func (egcd EthGasConsumeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	}
 
 	// we know that we have enough gas on the pool to cover the intrinsic gas
-	// set up the updated context to the evm Keeper
-	egcd.evmKeeper.WithContext(ctx)
```

Comment:
withContext also removed, any side effect?

Reply:
https://github.com/tharsis/ethermint/pull/729#issuecomment-971114131
You can have a look at the review guide, it's explained there.